net/relayd/src/opnsense/mvc/app/models/OPNsense/Relayd/Relayd.php

```diff
@@ -38,4 +38,19 @@
  */
 class Relayd extends BaseModel
 {
+    /**
+     * get, set, unset configuration state
+     * @param $action - null = get, true = set, false = unset
+     * @return bool
+     */
+    public function configChanged($status = null)
+    {
+        if ($status === null) {
+            return file_exists("/tmp/relayd.dirty");
+        } elseif ($status === true) {
+            return touch("/tmp/relayd.dirty");
+        } elseif ($status === false) {
+            return unlink("/tmp/relayd.dirty");
```

Comment:
@unlink

if we leave the error report on they will fill the crash reporter with spurious reports

Reply:
Ok, got it :smirk: